Conflicting types for 'write' at devmbboDirectStream.c

Compiling stream_2_7_7 for RTEMS and receiving the following error:
```bash
/afs/slac/package/rtems/4.9.4//host/i386_linux2x//bin/powerpc-rtems-gcc --pipe -B/afs/slac/package/rtems/4.9.4//target/rtems_p0/powerpc-rtems/beatnik/lib/ -specs bsp_specs -qrtems   -fasm   -mcpu=powerpc -D__ppc_generic -mmultiple -mstring                        -O2 -g -g -g   -Wall       -DMY_DO_BOOTP=NULL -DHAVE_MOTLOAD -DRTEMS_NETWORK_CONFIG_MBUF_SPACE=2048 -DRTEMS_NETWORK_CONFIG_CLUSTER_SPACE=5120     -I. -I../O.Common -I. -I. -I.. -I../../include/compiler/gcc -I../../include/os/RTEMS -I../../include                        -I/afs/slac/g/lcls/epics/R3.15.5-1.0/modules/asyn/R4.30-1.1.1/include -I/afs/slac/g/lcls/epics/base/R3.15.5-1.0/include/compiler/gcc -I/afs/slac/g/lcls/epics/base/R3.15.5-1.0/include/os/RTEMS -I/afs/slac/g/lcls/epics/base/R3.15.5-1.0/include       -I/afs/slac/package/rtems/4.9.4//host/i386_linux2x//powerpc-rtems/include -I/afs/slac/package/rtems/4.9.4//target/rtems_p0/ssrlApps_p3/powerpc-rtems/beatnik//include -I/afs/slac/package/rtems/4.9.4//target/rtems_p0/ssrlApps_p3/powerpc-rtems/beatnik//include -c ../devmbboDirectStream.c
../devmbboDirectStream.c:84: error: conflicting types for 'write'
/afs/slac.stanford.edu/package/rtems/vol16/host/i386_linux2x/bin/../lib/gcc/powerpc-rtems/4.3.2/../../../../../i386_linux26/powerpc-rtems/include/sys/unistd.h:166: error: previous declaration of 'write' was here
make[1]: *** [devmbboDirectStream.o] Error 1
make[1]: Leaving directory `/afs/slac.stanford.edu/g/lcls/vol8/epics/iocTop/users/slepicka/sandbox/git/streamdevice-git/src/O.RTEMS-beatnik'
make: *** [install.RTEMS-beatnik] Error 2

```

--- a/src/devmbboDirectStream.c
+++ b/src/devmbboDirectStream.c
@@ -81,7 +81,7 @@ static long initRecord (dbCommon *record)
    Thus make sure the record is never left in INVALID_ALARM status.
 */
 
-static long write(dbCommon *record)
+static long write_mbbo(dbCommon *record)
 {
     long status = streamWrite(record);
     if (record->nsev == INVALID_ALARM) record->nsev = MAJOR_ALARM;
@@ -102,7 +102,7 @@ struct {
     streamInit,
     initRecord,
     streamGetIointInfo,
-    write
+    write_mbbo
 };
 
 epicsExportAddress(dset,devmbboDirectStream);

